src/videojs-contrib-hls.js

```diff
@@ -48,6 +48,25 @@ Object.defineProperty(Hls, 'GOAL_BUFFER_LENGTH', {
   }
 });
 
+Object.defineProperty(Hls, 'BUFFER_LOW_WATER_LINE', {
+  get() {
+    videojs.log.warn('using Hls.BUFFER_LOW_WATER_LINE is UNSAFE be sure ' +
+                     'you know what you are doing');
+    return Config.BUFFER_LOW_WATER_LINE;
+  },
+  set(v) {
```

Comment:
I agree, though I would say in this simple setter case, there isn't any ambiguity. Also the other setters are this way.